### PR TITLE
ignore url checks to biorxiv

### PR DESCRIPTION
**Pull request by andreacosolo**
The static section URL check tries to send a GET request to biorxiv, (either directly or through doi.org), which fails because biorxiv doesn’t like these automated requests anymore.
Skip requests to biorxiv and ignore 403s that come from doi, so that the check gets green again.

### CODE REVIEW

File: chalicelib/checks/audit_checks.py [hunk 1]
```diff
@@ -425,6 +425,9 @@ def check_help_page_urls(connection, **kwargs):
         for url in urls:
             if url.startswith('mailto'):
                 continue
+            elif 'biorxiv.org'.lower() in url:
```
Comment (sbreiff) [inline chalicelib/checks/audit_checks.py:428]: should this be `elif 'biorxiv.org' in url.lower():`? biorxiv.org is already lowercase.

File: chalicelib/checks/audit_checks.py [hunk 2]
```diff
@@ -433,11 +436,12 @@ def check_help_page_urls(connection, **kwargs):
                 continue
             try:
                 request = requests.get(url.replace('&amp;', '&'), timeout=2)
-                if request.status_code not in [200, 412]:
+                if request.status_code == 403 and 'doi.org'.lower() in url:
```
Comment (sbreiff) [inline chalicelib/checks/audit_checks.py:439]: here too doi.org is already lowercase, should this be `url.lower():`?

**Review by sbreiff — COMMENTED** [review]
I commented on your use of `.lower()` in a couple lines. In addition, I see that the poetry.lock file has been updated - did you commit this on purpose? Usually we only update this when we need to update dependencies, but if the tests didn't pass until you committed this then it makes sense to update it.

File: chalicelib/checks/audit_checks.py (hunk 1 above)
Comment (andreacosolo) [reply chalicelib/checks/audit_checks.py]: That's what I meant, thanks for catching that!

**Comment by andreacosolo** [comment]
> I see that the poetry.lock file has been updated - did you commit this on purpose? Usually we only update this when we need to update dependencies, but if the tests didn't pass until you committed this then it makes sense to update it.

When running the check locally, it didn't work until I did `poetry update`. Since I saw that tests didn't pass, I was hoping that this would fix them. But it looks like test_get_s3_utils is still failing. @willronchetti any idea why?

**Pull request merged**